server/src/test/java/org/elasticsearch/indices/IndicesServiceTests.java

```diff
@@ -271,6 +271,8 @@ public void testDeleteIndexStore() throws Exception {
         ensureGreen("test");
     }
 
+    // NORELEASE This test need to be adapted for replicated closed indices
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/33888")
```

Comment:
same here

Reply:
Sorry it's not. I'll add a bullet point to #33888. The test will be fixed in a follow up pull request (code is ready)